pkg/util/crypt/crypt_dev.go

```diff
@@ -36,11 +38,27 @@ var (
 	ErrInvalidPassphrase = errors.New("no key available with this passphrase")
 )
 
+func getMaxLoopDevices() int {
```

Comment:
This `getMaxLoopDevices` is repeated in many locations - please instead define it once as an exported/Public function (probably in `pkg/util/loop` is a good place - and then call it from other places it is needed.